src/parserstate.c

```diff
@@ -204,7 +204,7 @@ static rbs_ast_comment_t *parse_comment_lines(parserstate *state, comment *com)
 
   return rbs_ast_comment_new(
     &state->allocator,
-    rbs_location_pp(&com->start, &com->end),
+    alloc_location(&state->allocator, (range){.start = com->start, .end = com->end}),
```

Comment:
I think we had spaces on these in other places. pls confirm

```suggestion
    alloc_location(&state->allocator, (range) { .start = com->start, .end = com->end }),
```

Reply:
Updated.